linkerd/proxy/api-resolve/src/metadata.rs

```diff
@@ -75,6 +79,7 @@ impl Metadata {
             authority_override,
             weight,
             http2,
+            is_zone_local: zone_locality,
```

Comment:
```suggestion
            is_zone_local,
```

nit: we could rename this parameter and rely on punning. it seems subjectively clearer to put an `is_..` prefix on a boolean too?

Reply:
Done, looks like this was a variable missed in an automated rename.